pkg/api/api.go

```diff
@@ -388,6 +388,11 @@ func (hs *HTTPServer) registerRoutes() {
 			annotationsRoute.Post("/graphite", reqEditorRole, bind(dtos.PostGraphiteAnnotationsCmd{}), Wrap(PostGraphiteAnnotation))
 		})
 
+		// The live handler will manage its own routes
+		if hs.Live != nil {
+			apiRoute.Any("/live/*", hs.LiveHTTPHandler)
```

Comment:
I tried using:
```suggestion
			apiRoute.Any("/live/*", Wrap(hs.LiveHTTPHandler))
```
but the circular imports to use any response helpers was too difficult to sort out